python/lsst/ts/cRIOpy/GUI/CompressorsPageWidget.py

```diff
@@ -0,0 +1,286 @@
+        )
+        dataLayout.addRow(
+            "Motor input", DataLabel(self.compressor.analogData, "motorInput")
+        )
+        dataLayout.addRow(
+            "Power consumption",
+            DataLabel(self.compressor.analogData, "compressorPowerConsumption"),
+        )
+        dataLayout.addRow(
+            "Volume percentage",
+            DataLabel(self.compressor.analogData, "compressorVolumePercentage"),
+        )
+        dataLayout.addRow(
+            "Volume", DataLabel(self.compressor.analogData, "compressorVolume")
+        )
+        dataLayout.addRow(
+            "Stage 1 output pressure",
+            DataLabel(self.compressor.analogData, "stage1OutputPressure"),
+        )
+        dataLayout.addRow(
+            "Line pressure", DataLabel(self.compressor.analogData, "linePressure")
+        )
+        dataLayout.addRow(
+            "Stage 1 output temperature",
+            DataLabel(self.compressor.analogData, "stage1OutputTemperature"),
+        )
+        dataLayout.addRow(
+            "Running hours", DataLabel(self.compressor.timerInfo, "runningHours")
+        )
+        dataLayout.addRow(
+            "Loaded hours", DataLabel(self.compressor.timerInfo, "loadedHours")
+        )
+        dataLayout.addRow(
+            "Lowest service counter",
+            DataLabel(self.compressor.timerInfo, "lowestServiceCounter"),
+        )
+        dataLayout.addRow(
+            "Run-On timer", DataLabel(self.compressor.timerInfo, "runOnTimer")
+        )
+        dataLayout.addRow(
+            "Loaded hours 50 percent",
+            DataLabel(self.compressor.timerInfo, "loadedHours50Percent"),
+        )
+
+        statusLayout.addRow(
+            "Read to start", OnOffLabel(self.compressor.status, "readyToStart")
+        )
+        statusLayout.addRow(
+            "Operating", OnOffLabel(self.compressor.status, "operating")
+        )
+        statusLayout.addRow(
+            "Start Inhibit", OnOffLabel(self.compressor.status, "startInhibit")
+        )
+        statusLayout.addRow(
+            "Motor start phase", OnOffLabel(self.compressor.status, "motorStartPhase")
+        )
+        statusLayout.addRow("Off load", OnOffLabel(self.compressor.status, "offLoad"))
+        statusLayout.addRow("On load", OnOffLabel(self.compressor.status, "onLoad"))
+        statusLayout.addRow("Soft stop", OnOffLabel(self.compressor.status, "softStop"))
+        statusLayout.addRow(
+            "Run on timer", OnOffLabel(self.compressor.status, "runOnTimer")
+        )
+        statusLayout.addRow("Fault", OnOffLabel(self.compressor.status, "fault"))
+        statusLayout.addRow("Warning", OnOffLabel(self.compressor.status, "warning"))
+        statusLayout.addRow(
+            "Service required", OnOffLabel(self.compressor.status, "serviceRequired")
+        )
+        statusLayout.addRow(
+            "Min. allowed speed",
+            OnOffLabel(self.compressor.status, "minAllowedSpeedAchieved"),
```

Comment:
Based on other parts, this might be:
```
OnOffLabel(self.compressor.status, "minAllowedSpeed"),
```

Reply:
Why? The XML variable is really called Achieved, Achieved was dropped in Label for brevity (Max. allowed speed achieved would be too long). Variable names in XML are taken from CompAir documentation (which I will upload to confluence..)